packages/schematics/update/package.json

```diff
@@ -13,11 +13,13 @@
   "dependencies": {
     "@angular-devkit/core": "0.0.0",
     "@angular-devkit/schematics": "0.0.0",
+    "@types/npm-package-arg": "^6.1.0",
```

Comment:
These type of dependencies are build time only and should be placed in the root package.json.

This should also fix the bazel build issue

Reply:
I see, moved the dependency to the root `package.json`. I also made a dev dependency, since I don't think it would necessary at runtime (though the way `scripts/` are loaded with a require extension, I think they might require types at runtime, but I don't think schematics are loaded in that fashion). `yarn bazel build //packages/schematics/update` now succeeds.

Just out of curiosity, why are build-time dependencies required in the root `package.json`? I would think each package would include all the dependencies necessary to build them. Is this required for Bazel to generate the NPM build targets?